memcache/memcache.go

```diff
@@ -868,28 +871,11 @@ func (c *Client) processMetaSet(rw *bufio.ReadWriter, item *MetaSetItem, cb func
 	return fmt.Errorf("memcache: unexpected response line from ms: %q", string(response))
 }
 
-func getMemCacheErrorFromResponse(response []byte) error {
```

Comment:
I believe this is good to have. If memcache send server error then we can clearly tell what the error is with error message from memcache.

Reply:
I removed this because I wasn't using this. My `processMetaSet` method handles all other responses on line 871 when none of the case blocks at the bottom of the method are matched to by raising an error with the message given with this line `return fmt.Errorf("memcache: unexpected response line from ms: %q", string(response))`. We can add this through your changes because you are using it.